From 858276f1707468e5f55c37287cff9e8874812ae7 Mon Sep 17 00:00:00 2001
From: "Adam C. Emerson" <aemerson@redhat.com>
Date: Sat, 12 Mar 2016 11:25:29 -0500
Subject: [PATCH] gitignore: Add one more file

So git will stop claiming there's New Content here when we build Ceph.

Signed-off-by: Adam C. Emerson <aemerson@redhat.com>
---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 99b1759..7f07af9 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 .deps/
 .dirstamp
 src/radosgw-civetweb.o
+src/libcivetweb_la-civetweb.o
 
 civetweb
 libcivetweb.a
